Libs/Visualization/VTK/Widgets/ctkVTKPiecewiseFunction.h

```diff
@@ -39,30 +39,31 @@ class CTK_VISUALIZATION_VTK_WIDGETS_EXPORT ctkVTKPiecewiseFunction: public ctkTr
   Q_OBJECT;
   QVTK_OBJECT;
 public:
+  ctkVTKPiecewiseFunction(QObject* parent = 0);
   ctkVTKPiecewiseFunction(vtkPiecewiseFunction* piecewiserFunction,
                               QObject* parent = 0);
   virtual ~ctkVTKPiecewiseFunction();
 
-  virtual ctkControlPoint* controlPoint(int index)const;
-  virtual QVariant value(qreal pos)const;
-  virtual int count()const;
-  virtual bool isDiscrete()const;
-  virtual bool isEditable()const;
+  Q_INVOKABLE virtual ctkControlPoint* controlPoint(int index)const;
+  Q_INVOKABLE virtual QVariant value(qreal pos)const;
+  Q_INVOKABLE virtual int count()const;
+  Q_INVOKABLE virtual bool isDiscrete()const;
+  Q_INVOKABLE virtual bool isEditable()const;
```

Comment:
Property for is...() maybe for count() as well. Check what is usually done in other CTK classes.